common/lib/xmodule/xmodule/tests/test_services.py

```diff
@@ -0,0 +1,74 @@
+"""
+Tests for SettingsService
+"""
+
+import ddt
+import mock
+from unittest import TestCase
+
+from django.conf import settings
+from django.test.utils import override_settings
+
+from xblock.runtime import Mixologist
+from lms.djangoapps.lms_xblock.runtime import SettingsService
```

Comment:
This line is causing a test failure. I think you need to import it from `xmodule.services` instead of `lms_xblock`.